addons/calendar/tests/test_mail_activity_mixin_model.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class CalendarTestActivity(models.Model):
+    """ This model can be used to test activities
+    """
+    _description = 'Activity Model'
+    _name = 'calendar.test.activity'
```

Comment:
Crac. Cannot do that on a classic addon, as it implies creating a table and a model people won't use in production environment. You have to use a model inheriting from activity mixin available in calendar, probably res.partner. What you can do is create a test record that is a res.partner and your test should still be ok (maybe give partner_manager group to your test user so that he can write on partner records).

The other way would be to do a test_calendar module depending on test_mail but I think this would be a bit too much. Maybe later if we add more addon-specific behavior to mail or activities.